subprojects/docs/src/docs/release/notes.md

```diff
@@ -94,6 +94,10 @@ See [the userguide](userguide/more_about_tasks.html#sec:property_file_normalizat
 
 [//]: # (TODO context and overview of improvements)
 
+#### Composite builds
+
+Starting with this release, [composite builds](userguide/composite_builds.html) are fully supported with the configuration cache and thus will no longer be reported as problems.
```

Comment:
"will no longer be reported as problems" sounds a bit weird to me. I think just removing this part of the sentence would be fine.

Reply:
Thanks for the feedback, @pioterj. Done.